nearswap/src/deposit.rs

```diff
@@ -233,10 +244,10 @@ impl DepositV1 {
 
     pub(crate) fn remove(&mut self, token: &AccountId, amount: u128) {
         if let Some(x) = self.tokens.get_mut(token) {
-            assert!(*x >= amount, ERR13_NOT_ENOUGH_TOKENS_DEPOSITED);
+            assert!(*x >= amount, "{}", ERR13_NOT_ENOUGH_TOKENS_DEPOSITED);
             *x = *x - amount;
         } else {
-            panic!(ERR13_NOT_ENOUGH_TOKENS_DEPOSITED);
+            panic!("{}", ERR13_NOT_ENOUGH_TOKENS_DEPOSITED);
```

Comment:
do we really need to put `"{}"` in panic?

Reply:
![strf](https://user-images.githubusercontent.com/26249903/119825442-92ea1e80-bf14-11eb-9631-a20da9da9e64.png)

yes linter shows this warning

Reply:
this one for panic
![kjsf](https://user-images.githubusercontent.com/26249903/119825754-ec524d80-bf14-11eb-8db7-cbb09e3bbc76.png)